01_fib/shahriys/main.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
Should shahriy's two files be in you PR?

Reply:
No, i dont understand how it got added